Makefile

```diff
@@ -86,29 +87,26 @@ test-unit: ## Run the unit tests
 	$(Q)go test -count=1 -short $(UNIT_TEST_DIRS)
 
 test-e2e: ## Run the e2e tests
-	$(Q)go test -count=1 ./test/e2e
```

Comment:
I realize this probably needs to be something like `go run github.com/onsi/ginkgo/ginkgo ...` or using a $(GINKGO) variable that runs that command, and updating it here to use the variable.

Reply:
Ah, yep that was the missing piece!